akka-actor/src/main/java/akka/annotation/ApiMayChange.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.annotation;
+
+import java.lang.annotation.*;
+
+/**
+ * Marks APIs that are meant to evolve towards becoming stable APIs, but are not stable APIs yet.
+ * 
+ * Evolving interfaces can change from one feature release to another release (i.e. 2.1 to 2.2) without up-front notice.
```

Comment:
It doesn't have to wait until 2.4.0 for promotion.

Thing comes in 2.3.4, we iterate in 2.3.5, 2.3.6, 2.3.7, and promote it in 2.3.8.

During 2.3.4 - 2.3.7 we do best effort in keeping it source compatible (exceptions may occur). In 2.3.8 it might not be source compatible, e.g. removing deprecated or doing final refactoring.